src/components/forms/Toggle.tsx

```diff
@@ -445,29 +521,27 @@ export function Radio() {
         a.align_center,
         a.rounded_full,
         t.atoms.border_contrast_high,
+        a.transition_color,
         {
           borderWidth: 1,
-          height: 24,
-          width: 24,
+          height: 25,
```

Comment:
Designs have this at 26px, and I asked Darrin if it should match the switch at 28px.